homework-submission/todo-app/src/Components/List.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from "react";
+import '../App.css';
+import Task from './Task';
+import Checkbox from './Checkbox';
+import { observer, inject } from 'mobx-react';
+
+@inject('TasksStore')
+@observer
+
+class List extends Component {
+    render() {
+        const { TasksStore } = this.props;
+        return (
+            <div className='List'>
+                <ul>
+                    {TasksStore.todos.map((todo, id) => (
```

Comment:
What is the value of `id`?
`id` is for identifying, while index for what it means, don't we want to identify ? if yes 😃 then we may use `todo.id` instead of `id` ?